src/client/src/components/MealList.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import mealsAPI from '../api/meals';
+
+export default function MealList() {
+  const [meals, setMeals] = useState([]);
+
+  useEffect(() => {
+    async function fetchData() {
+      const res = await mealsAPI.getAll();
+      console.log(res);
+      setMeals(res);
+    }
+    fetchData();
+  }, []);
+  return (
+    <div className='meals-container'>
+      {meals.map((meal) => (
```

Comment:
Consider a case where the meals are not (yet) available, so that this does no throw an error and shows something meaningful to the user 💡